QKSMS/src/main/java/com/moez/QKSMS/common/DialogHelper.java

```diff
@@ -70,4 +76,23 @@ public void onClick(View v) {
                 }).setNegativeButton(R.string.cancel, null)
                 .show(context.getFragmentManager(), QKDialog.CONFIRMATION_TAG);
     }
+
+    public static void showDeleteFailedConversationDialog(final MainActivity context, final List<Long> threadIds) {
```

Comment:
Actually this method isn't necessary. There isn't an option to delete failed messages in multi-select mode, so we don't need a method that accepts a list of threadIds. Only the first one with a single threadId is necessary